RULES/core/main.go

```diff
@@ -26,6 +26,7 @@ type generatorInput struct {
 	CompletionsOnly bool
 	RunArgs         []string
 	TestArgs        []string
+	Layout          string
```

Comment:
What happens when we have a workspace with mixed layout?

Reply:
Removed, this is not used